src/libs/actions/IOU.ts

```diff
@@ -1194,6 +1198,72 @@ function getFieldViolationsOnyxData(iouReport: OnyxTypes.Report): SetRequired<On
     };
 }
 
+function buildOnyxDataForTestDriveIOU(
+    transaction: OnyxTypes.Transaction,
+    iouOptimisticParams: MoneyRequestOptimisticParams['iou'],
+    chatOptimisticParams: MoneyRequestOptimisticParams['chat'],
+): [OnyxUpdate[], OnyxUpdate[], OnyxUpdate[]] {
```

Comment:
```suggestion
function buildOnyxDataForTestDriveIOU(testDriveIOUParams: BuildOnyxDataForTestDriveIOUParams): OnyxData {
```

Let's write `BuildOnyxDataForTestDriveIOUParams` type for consistency with other functions. Also the return type is better handled with `OnyxData` as an object instead of an array.